### PR TITLE
Added metadata field

### PR DESCRIPTION
**Pull request by PakDLiu**
Added metadata field
Updated release history

### CODE REVIEW

File: azext_hanaonazure/custom.py [hunk 1]
```diff
@@ -280,10 +280,17 @@ def create_providerinstance(
       monitor_name,
       provider_instance_name,
       provider_instance_type,
-      provider_instance_properties):
+      provider_instance_properties,
+      provider_instance_metadata=None):
```
Comment (persiaAziz-zz) [inline azext_hanaonazure/custom.py:284]: What about updating the caller where this function gets called?

Comment (PakDLiu) [reply azext_hanaonazure/custom.py:284]: The framework magically call this function and can detect the parameters and their default.
Just updating the signature of this method is enough to change the command. I know, magic

**Review by tniek — APPROVED** [review]
LGTM

**Review by persiaAziz-zz — APPROVED** [review]
+1

**Pull request merged**